purchase_order_type/models/purchase_order.py

```diff
@@ -11,17 +11,14 @@ class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
 
     def _default_order_type(self):
-        return self.env["purchase.order.type"].search(
-            [("company_id", "in", [False, self.env.company.id])], limit=1,
-        )
+        return self.env["purchase.order.type"].search([], limit=1)
```

Comment:
You have to explain why this, and it seems incorrect at first sight.

Reply:
Purchase order type model have the same rule (https://github.com/OCA/purchase-workflow/blob/13.0/purchase_order_type/security/security.xml) and IMO is not necessary to apply these domain.

Reply:
You don't know if the operation is done in sudo mode, so please keep it.

Reply:
This comment still applies

Reply:
Sorry, it is not necessary to apply a domain to filter by company because this domain is already defined as a rule of the model.

Reply:
Such domain doesn't apply here. It's just for UI.